common/network-yarn/src/main/java/org/apache/spark/network/yarn/YarnShuffleService.java

```diff
@@ -118,10 +121,10 @@ public class YarnShuffleService extends AuxiliaryService {
   private static final String SPARK_AUTHENTICATE_KEY = "spark.authenticate";
   private static final boolean DEFAULT_SPARK_AUTHENTICATE = false;
 
-  private static final String RECOVERY_FILE_NAME = "registeredExecutors.ldb";
-  private static final String SECRETS_RECOVERY_FILE_NAME = "sparkShuffleRecovery.ldb";
+  private static final String RECOVERY_FILE_NAME = "registeredExecutors";
```

Comment:
is this going to cause any compatibility issues?  What about upgrading and/or changing DB?   If I just upgrade to this version of spark shuffle during rolling upgrade, I assume it won't find the old file now. Seems safer to keep the .ldb when using leveldb, unless that is handled below and I haven't gotten to it yet.

Reply:
Line 255 will add the file suffix. If LevelDB is used, the suffix is `.ldb`, Is this ok?

Reply:
thanks, missed that.  Yeah .ldb.. we want to keep it the same

Reply:
> thanks, missed that. Yeah .ldb.. we want to keep it the same

Hmm...do you mean that if RocksDb is used, the file is also called `registeredExecutors .ldb`?

Reply:
no, sorry, level db should keep .ldb for backwards compatible.  If someone switches to rocksdb, I assume its going to break the upgrade path with support for rolling restart because the database name would change and all the previous data stored in leveldb would not be read by rocksdb.   personally I think this is acceptable we just need to document it somewhere.  I guess someone could convert ldb to rocksdb on startup but I think we could wait for someone to request this.

Reply:
Thanks for your explanation,  I will remember add document to describe this after the implementation of RocksDB is completed

> I guess someone could convert ldb to rocksdb on startup but I think we could wait for someone to request this.

This is a very interesting requirement ~